hyperopt/fmin.py

```diff
@@ -9,6 +9,7 @@
 import os
 import sys
 import time
+import timeit
```

Comment:
given you only use default_timer, what do you think about changing this to `from timeit import default_timer as timer` ?

Reply:
Agree